evm-adapters/src/sputnik/mod.rs

```diff
@@ -28,6 +28,7 @@ use sputnik_evm::executor::stack::PrecompileSet;
 /// the live chain data returned by the provider.
 pub async fn vicinity<M: Middleware>(
     provider: &M,
+    passed_chain_id: Option<u64>,
```

Comment:
rename `override_chain_id`?

Reply:
it looks like this is always called with `None`?

Reply:
> it looks like this is always called with `None`?

How so? If you check the Drai tests where we pass `--chain-id 99` in the integration tests it is `Some(99)` and the tests are passing

Reply:
ah right, overlooked the call here
https://github.com/gakonst/foundry/pull/603/files#diff-987f8b2fa39143d16bbaaeff2eacb089e0a9c2297e4ff42a79cbb0edc23bda9cR138-R143